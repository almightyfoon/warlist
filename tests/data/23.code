c85b1b_-0pex5m5m4P4F4F4H4H4Hg4g45g5f5fgs__4w4M4M4z4H4HfF4_4U4WaGgs