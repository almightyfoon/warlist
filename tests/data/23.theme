{"allowed":[46,47,75,190,207,233,234,235,236,240,320,323,334,335,336,337,339,340,348,350,388,391,401,424,425,426,455,456,457,458,460,466,477,478,479,481,489,490,491,492,493,494,498,499,500,501,503,504,505,507,508,509,510,511,512,513,518,519,520,522,523,524,530,531,533,534,536,537,538,539,541,547,548,562,566,567,570,577,579,580,581,582,639,655,670,683,688,696,700,701,726,785,822,2066,3021,3031,3032,3036,3059,3060,3061,3062,3123,3140,3176,3177,3178,3179,4005,4024,4034,4038,4040,6001,6023,6094,6095,6097,6102,6105,6123,6124,6160,6187,6294,6302,6304,6327,6393,6403,6404,6405,6406,6407,6408,6412,6414],"required":[4,5,6,12,14,19,20,21,26,27,29,30,31,46,47,71,72,74,75,76,77,78,79,81,83,85,86,87,88,90,91,92,93,94,100,101,103,105,108,110,112,116,118,120,121,122,125,126,127,140,167,172,173,174,201,206,213,217,224,233,234,235,236,242,245,246,252,254,255,256,257,258,259,260,261,262,263,264,265,267,269,271,272,273,301,302,314,318,319,320,321,322,323,324,325,326,332,334,335,336,337,338,356,363,364,380,382,383,402,406,408,409,410,411,413,414,416,417,418,419,455,456,457,471,473,474,499,500,501,503,508,509,512,520,522,523,528,530,531,532,533,534,535,536,537,538,539,540,541,578,579,580,588,589,593,594,595,596,597,598,599,600,619,620,626,627,634,635,637,639,640,642,643,644,646,655,683,684,685,686,687,688,689,696,726,727,777,782,783,785,799,804,805,810,811,812,2002,2003,2004,2005,2007,2008,2009,2031,2032,2033,2041,2045,2066,3021,3022,3023,3024,3025,3042,3043,3060,3061,3062,3065,3066,3067,3068,3070,3095,3097,3098,3100,3101,3119,3120,3121,3127,3129,3130,3132,3133,3139,3140,3143,3144,3162,3173,3176,3177,3178,4022,4025,4026,4035,4036,4037,4038,4040,6017,6019,6026,6028,6029,6031,6032,6033,6034,6035,6036,6052,6061,6067,6069,6096,6124,6131,6132,6138,6156,6160,6161,6162,6165,6177,6187,6244,6250,6272,6273,6274,6281,6285,6286,6287,6288,6300,6301,6302,6345,6356,6393,6399,6403,6404,6405,6414],"free":[1,11,23,57,96,106,109,124,129,131,134,135,138,141,143,144,145,146,147,149,150,152,156,158,163,165,169,175,177,181,183,184,185,186,187,188,189,192,193,194,195,196,197,198,200,202,204,205,207,208,210,211,215,216,219,222,238,239,240,241,244,268,276,277,285,291,292,296,297,298,299,300,305,307,311,312,313,315,343,346,358,359,360,361,362,367,368,369,372,373,374,375,376,379,381,387,399,403,404,405,407,412,420,421,422,423,427,429,436,441,447,452,454,459,460,461,462,463,464,465,467,468,476,477,482,483,484,485,486,487,488,505,514,515,517,519,525,526,527,543,548,550,558,563,564,566,567,568,569,570,571,581,582,602,607,610,613,614,615,616,617,618,623,629,633,636,647,649,652,653,654,656,657,658,672,698,701,703,774,776,781,784,802,809,821,2011,2012,2024,2025,2026,2027,2040,2042,2043,2044,2048,2052,2053,2067,2068,3026,3027,3028,3029,3030,3031,3038,3039,3040,3041,3058,3059,3093,3103,3104,3134,3137,3138,3142,3154,3155,3157,3159,3166,3169,3172,3175,3180,3183,3187,3188,3189,4001,4004,4010,4011,4012,4021,4024,4027,4028,4029,4031,4032,4033,6030,6044,6045,6046,6048,6049,6050,6051,6063,6066,6070,6071,6072,6076,6095,6098,6099,6101,6104,6105,6106,6134,6137,6140,6154,6155,6167,6168,6169,6171,6185,6186,6189,6190,6193,6202,6203,6210,6212,6214,6215,6216,6217,6236,6245,6262,6263,6265,6266,6276,6280,6290,6292,6294,6295,6296,6297,6303,6304,6305,6306,6308,6309,6311,6368,6369,6377,6382,6383,6389,6392,6397,6398,6402,6409,6410,6411,6415,6416,6418,6419,6420]}