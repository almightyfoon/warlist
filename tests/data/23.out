https://conflictchamber.com/#c85b1b_-0pexjGjGjFjEjE4H4H4Hg4g45g5f5fgs__4w4M4M4z4H4HfF4_4U4Wi7gs

Circle Army - 68 / 75 points
[Theme] The Bones of Orboros

[Thorle 1] Bradigus Thorle the Runecarver [+27]
 - Wold Guardian [16]
 - Wold Guardian [16]
 - Woldwarden [14]
 - Woldwatcher [8]
 - Woldwatcher [8]
 - Woldwyrd [9]
 - Woldwyrd [9]
 - Woldwyrd [9]
Blackclad Stoneshaper [0(3)]
Blackclad Stoneshaper [0(3)]
Blackclad Wayfarer [0(4)]
Shifting Stones [3]
Shifting Stones [3]
Bunker (Objective) [0]

Circle Army - 75 / 75 points

[Mohsar 1] Mohsar the Desertwalker [+27]
 - Pureblood Warpwolf [17]
 - Pureblood Warpwolf [17]
 - Wild Argus [7]
 - Woldwyrd [9]
 - Woldwyrd [9]
Swamp Gobber Chef [1]
Tharn Bloodweavers [8]
Wolves of Orboros (max) [11]
 - Wolves of Orboros Chieftain & Standard [4]
Celestial Fulcrum [19]
Bunker (Objective) [0]
